package.json

```diff
@@ -10,18 +10,21 @@
     "@babel/core": "^7.12.3",
     "@babel/node": "^7.12.1",
     "@babel/preset-env": "^7.12.1",
+    "@types/node": "^14.14.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
```

Comment:
Express should be added in "dependencies" not in "devDependencies", it required to run code on production